Logger/FileLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string path;
+
+        public FileLogger(string filePath)
+        {
+            path = filePath;
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            if(!(path is null) && File.Exists(path))
+            {
+                string output = DateTime.Now.ToString("MM/dd/yyyy hh:mm:ss tt") + " " + ClassName + " " + logLevel + ": " + message + Environment.NewLine;
```

Comment:
String interpolation can make this simpler
```suggestion
                string output = $"{DateTime.Now:MM/dd/yyyy hh:mm:ss tt} {ClassName} {logLevel}: {message}{Environment.NewLine}";
```